From 48c70d1e1becb929478835222af144c2d3ad0266 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 12 Jan 2024 09:51:22 +0100
Subject: [PATCH 0849/1141] arm64: dts: st: add power supplies to lvds on
 stm32mp257f-ev1 board

LVDS need two power supplies: v1v8 & vddcore

Change-Id: I6511a5721c1f7c9cfbe26ebd0c56be1506c14864
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/350857
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index dcad3cf2c65e..6be9fa6d975e 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -427,6 +427,8 @@ ltdc_ep1_out: endpoint@1 {
 };
 
 &lvds {
+	vdd-supply =  <&scmi_vddcore>;
+	vdda18-supply = <&scmi_v1v8>;
 	status = "okay";
 
 	ports {
-- 
2.39.2

